Sources/Config.php

```diff
@@ -769,7 +769,7 @@ class Config
 				 * Path to the Packages directory.
 				 */
 				END,
-			'default' => 'dirname(__FILE__) . \'/Packages\'',
+			'default' => '__DIR__ . \'/Packages\'',
```

Comment:
This is acceptable, but only if the same change is made in ./other/Settings.php.

Reply:
All references to dirname(__FILE__) changed to __DIR__ in Settings.php & Settings_bak.php.